libs/ui/src/lib/tabs/tabs.component.html

```diff
@@ -0,0 +1,76 @@
+<div class="cvi-tabs__list" role="tablist">
+  <button
+    *ngFor="let tab of allTabs; let i = index"
+    dataAttribute="tab-button-{{ i }}"
+    [ngClass]="{ active: isTabSelected(i), underlined: isTabSelected(i) }"
+    (click)="makeActive(i)"
+    type="button"
+    #tabButton
+    id="{{ baseId }}-tab-{{ i }}"
+    [attr.aria-selected]="isTabSelected(i)"
+    role="tab"
+    [attr.tabindex]="isTabSelected(i) ? null : '-1'"
+    [attr.aria-controls]="baseId + '-tabpanel-' + i"
+    class="cvi-tabs__tab-btn"
+  >
+    {{ tab.title }}
+  </button>
+</div>
+<div class="cvi-tabs__list-mobile" role="menu">
+  <button
+    role="button"
+    class="cvi-tabs__mobile-button"
+    type="button"
+    id="{{ baseId }}-tab-button}"
+    aria-expanded="false"
+    (click)="handleArrowButtonClick($event)"
+  >
+    {{ getActiveTabTitle() }}
+    <cvi-ng-icon
+      [class]="'cvi-tabs__icon'"
+      [name]="isOpen ? 'arrow_up_alt' : 'arrow_down_alt'"
+      [svgClass]="'cvi-tabs__arrow-icon'"
+    >
+    </cvi-ng-icon>
+  </button>
+</div>
+<div
+  *ngIf="isOpen"
+  dataAttribute="tabs-popup"
+  class="cvi-tabs__dropdown-popup"
+  #popup
+>
+  <div class="cvi-tabs__dropdown">
+    <ul
+      role="listbox"
+      class="cvi-tabs__dropdown-container"
+      (closed)="close()"
+      role="menu"
+      (keydown)="handleKeyboardEvent($event)"
+    >
+      <button
+        *ngFor="let tab of allTabs; let i = index"
+        class="cvi-tabs__dropdown-item"
+        dataAttribute="tab-button-{{ i }}"
+        (click)="makeActive(i)"
+        id="{{ baseId }}-tab-{{ i }}"
```

Comment:
These buttons (appearing on mobile) have the same `id` as tab buttons (appearing on desktop). Since we now manage what's hidden via CSS, both happen to exist in DOM at the same time. It's better to make them unique to avoid our accessibility checker complaining about non-unique ids: 
```suggestion
        id="{{ baseId }}-dropdown-button-{{ i }}"
```
![image](https://github.com/e-gov/cvi/assets/1833029/ad38198c-8595-482c-af3d-8afdda6eec89)